### DISCUSSION TITLE
Question about unresolved AnimationClip paths

### DISCUSSION THREAD

**Discussion by Zbuffers** [AssetRipper/AssetRipper#1157] [General]
Hello ;)
In recent versions, I noticed that when an animation path is not resolved, it is written using a HEX representation of its CRC32.
example:  
path_0xC18DE03C_TluhMWN
When doing a comparison with the CRC32 paths in the genericBindings  section,
I can find a CRC32 path:
3247300668 

so, 0xC18DE03C is the HEX of 3247300668. that's fine.

What is the suffix ?  (lthe cryptic string after the '_' char) ?,
In my example, what is the '_TluhMWN' part ?

Unrelated side question: Why is the static mesh separation disabled ?

Anyway, keep up the great work !
Haapy new year ;)
Kind regards,
Z

**Comment by ds5678** [comment]
The cryptic section is the 7 characters necessary for the string to have the same hash as the original.

Static mesh separation was removed after I made substantial improvements for the premium edition.
